clusterloader2/pkg/prometheus/manifests/0prometheus-operator-deployment.yaml

```diff
@@ -49,6 +50,9 @@ spec:
           readOnlyRootFilesystem: true
       nodeSelector:
         kubernetes.io/os: linux
+        {{if ne $PROMETHEUS_NODE_SELECTOR ""}}
```

Comment:
Nit: Do we need to if for != "".

It looks like simply writing
```
nodeSelector:
  {{$PROMETHEUS_NODE_SELECTOR}}
```

will work the same way.